motr/st/utils/motr_dgmode_io_st.sh

```diff
@@ -34,9 +34,9 @@ m0t1fs_dir="$motr_st_util_dir/../../../m0t1fs/linux_kernel/st"
 . $motr_st_util_dir/motr_local_conf.sh
 . $motr_st_util_dir/motr_st_inc.sh
 
-N=3
-K=3
-S=3
+N=6
+K=2
+S=0
 P=15
```

Comment:
[Suggestion]
Ca we have N, K, S as 4 + 2 + 0 , considering we are targeting LDR r2 with 4 + 2 + 0.

Reply:
6+2+0 is fine for ST's, we will be adding motr ST's in Automated jenkins tests.

Reply:
okay.